.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 service/kubernetes-vault
-init-container/kubernetes-vault-init
\ No newline at end of file
+init-container/kubernetes-vault-init
+.idea/
```

Comment:
Can you remove the entry for `.idea/`? IDE specific ignores should go into your global `.gitignore` in my option 😄